nautobot/ipam/models.py

```diff
@@ -960,6 +961,21 @@ def get_utilization(self):
         return UtilizationData(numerator=numerator_set.size, denominator=denominator)
 
 
+@extras_features("graphql")
+class PrefixLocationAssignment(BaseModel):
+    # Using `related_name="+"`` because prefix is the parent of this through model and can only be accessed by Prefix from its `locations` field.
+    # i.e Prefix.objects.filter(locations__in=...), Prefix.objects.first().locations etc.
+    prefix = models.ForeignKey("ipam.Prefix", on_delete=models.CASCADE, related_name="+")
```

Comment:
Still feels like we should have `related_name="location_assignments"` here instead - what am I missing?

Reply:
The identical outcomes would be obtained by doing this 
```python
prefix_instance.locations.all()
```
and

```
prefix_instance. location_assignments.all()
```

I am unsure why something should be accessible in two different ways.

Reply:
`prefix.locations` would be a set of Location objects; `prefix.location_assignments` would be a set of `PrefixLocationAssignment` issues, right?